test/bridge.test.ts

```diff
@@ -344,7 +381,6 @@ describe('dynamic paths', () => {
         url === '/foo/public.svg'
       ).toBeTruthy()
     }
-    await expectNoClientErrors('/foo/assets')
```

Comment:
shouldn't we keep testing these with `expectNoClientErrors`?

Reply:
I should keep the test.
However, there seems to be a bug and it will need to be fixed later.

Reply:
okay! let's keep them, but commented out then 🙏

Reply:
Thank you! I already commented out!
https://github.com/nuxt/bridge/pull/1141/commits/9b704761d6fbaef4cae3daef660ce24be0c00c6f